### ISSUE TITLE
feature request: bulk import / paste list of Session IDs

### ISSUE THREAD

**Issue by jdiercks** [oxen-io/session-android#1042]
When onboarding a new person to a team, it would be good to get them connected to everyone efficiently. Adding each person one at a time is tedious. If a list of Session IDs could be imported or pasted in one shot, it could save a lot of time / effort.

**Comment by slrslr** [comment]
Yes, accepted separator may be newline, comma, semicolon or space.

If IDs would be export including customized nicknames, it may be privacy issue for the contacts if the export is leaked.

Later if Session introduce multiple IDs for one person and user labeled groups of contacts (needed), it can be handy to export only certain group of certain ID (like work ID with real name and contact group named "work").

**Comment by oxenreport** [comment]
mass id input fields<3 like a photo upload, but for bulk id invitation
